lib/matplotlib/widgets.py

```diff
@@ -2536,19 +2543,28 @@ def _set_active_handle(self, event):
         e_idx, e_dist = self._edge_handles.closest(event.x, event.y)
         m_idx, m_dist = self._center_handle.closest(event.x, event.y)
 
+        contains = self._contains(event)
```

Comment:
AFAICS this is only used once, so does not need a variable and can be inlined below.

Reply:
👍